app/src/org/commcare/activities/CommCareActivity.java

```diff
@@ -125,8 +125,10 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         FragmentManager fm = this.getSupportFragmentManager();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.GINGERBREAD_MR1) {

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/8eba354c593b70275ff43733ae020a7bc8ce7196/app/src/org/commcare/activities/CommCareActivity.java#L128"></span>Unnecessary; SDK_INT is always &gt;= 16</td>
    </tr>
  </tbody>
</table>